lib/rules/declaration-block-no-shorthand-property-overrides/__tests__/index.mjs

```diff
@@ -208,7 +208,6 @@ testRule({
 			message: messages.rejected('font-variant', 'font-variant-caps'),
 			line: 1,
 			endLine: 1,
```

Comment:
[suggest] In addition, it'd be great if you could add `column`/`endColumn` like other cases 🙏🏼 

```suggestion
			line: 1,
			column: 36
			endLine: 1,
			endColumn: 48
```